docs/schedule-key-swap.md

```diff
@@ -0,0 +1,122 @@
+:::
+
+## Description
+
+The schedule coldkey swap feature works like this:
+
+- The schedule coldkey swap feature enables you to only schedule the actual swapping of your TAO funds from your old potentially compromised coldkey to a new coldkey. 
+- When you use this feature, it will not actually swap your TAO funds. It will only schedule the swap event. 
+- All scheduled coldkey swaps will be executed on-chain when the mainnet opens for normal operations on Thursday July 11, 5 PM US Eastern.
+
+## Do not use this feature if
+
+You may not need to use this feature. Do not use this feature if any of the following are true for you:
+
+- If you never used `btcli`, regardless of the Bittensor version, then do not use this feature. Your funds are safe. 
+- If you never used Bittensor version 6.12.2, then do not use this feature. Your funds are safe. 
+- If you only used any wallet browser extension or only a wallet application such as below, then your funds are safe:
+  - The Bittensor Wallet (for Chrome or iOS)
+  - SafeWallet, Talisman, Polkadot Vault, Tensor Wallet, Nova Wallet, polkadot.js
+  - Websites that utilize polkadot.js such as Taostats.io for staking unstaking
+- If you hold your TAO only on exchanges, then your funds are safe. 
+- If you installed Bittensor 6.12.2 not via PyPi package but directly from GitHub source, then do not use this feature. **Only the PyPi package of Bittensor 6.12.2 is malicious.**
+
+### Use this feature only if 
+
+Use this feature ONLY: 
+
+- If you downloaded the Bittensor PyPi package for the Bittensor version 6.12.2 and then performed any of the below operations.
+
+  **Impacted btcli 6.12.2 operations**
+
+  ```bash
+  btcli stake add
+  btcli stake remove
+  btcli wallet transfer
+  btcli root delegate
+  btcli root undelegate
+  btcli root set_take
+  btcli subnet register
+  ```
+
+:::important reminder
+If you installed Bittensor 6.12.2 not via PyPi package but directly from GitHub source, then do not use this feature. Only the PyPi package of Bittensor 6.12.2 is malicious.
+:::
+
+## Installing the schedule coldkey swap feature
+
+The schedule coldkey swap feature is available only at the below specified subtensor and bittensor repo branches:
+
+- **Subtensor repo**: Make sure to checkout the tag `specify release tag` on the Subtensor repo `https://github.com/opentensor/subtensor`.
+- **Bittensor repo and branch**: 
+    1. First make sure you do `git pull master` to pull the latest changes from the Bittensor repo `https://github.com/opentensor/bittensor`. 
+    2. Next, switch to `release/7.3.0` branch. 
+    3. Then install Bittensor by running the below command:
+    ```bash
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/opentensor/bittensor/release/7.3.0/scripts/install.sh)"
+    ```
+    4. Confirm that you have the correct Bittensor version installed by running the below command: 
+    ```bash
+    btcli --help
+    ```
+    You should see the version `7.3.0` in the output.
```

Comment:
Skunked again!  It still shows `7.2.0`.

Part of the install script says:

`From https://github.com/opentensor/bittensor`
 `* branch              master     -> FETCH_HEAD`
`Already on 'master'`
`Your branch is up to date with 'origin/master'.`

Not sure this is relevant.